src/Runner/PhpInterpreter.php

```diff
@@ -12,6 +12,11 @@
 {
 
 	/**
+	 * @param  string
+	 */
+	function addArguments($args);
```

Comment:
I added this method to the interface because of former `$arguments` property. The `addPhpIniValue()` would solve it. Maybe in `addPhpIniValue('name=value')` format.